config/gulp/javascript.js

```diff
@@ -70,6 +70,7 @@ gulp.task('eslint', function (done) {
 
   return gulp.src([
       'src/js/**/*.js',
+      '!src/js/**/_*.js',
```

Comment:
following the pattern of the sass example did you mean for this to be `!src/js/_**/*.js`?

Reply:
For the JS file, I tried leaving it in the  `src/js/components/` folder and added an underscore to the file like this:  `src/js/components/_chart.js`. The Sass files all have underscores already which is why I moved it into the ``src/scss/_experimental` folder.